app.rb

```diff
@@ -20,7 +34,7 @@ class Airbnb < Sinatra::Base
     if User.email_already_exists?(params[:Email]) == true
       redirect('/register') # Sinatra flash will have to be added later
     end
-    User.create(params[:Email], params[:Username], params[:Password])
+    session[:user] = User.create(params[:Email], params[:Username], params[:Name], params[:Password])
     redirect('/spaces')
   end
 
```

Comment:
Style - consistency in type case used, i.e use lowercase in above for email, name and password?